### DISCUSSION TITLE
Show features in lower zoom levels

### DISCUSSION THREAD

**Discussion by amiria703** [protomaps/basemaps#272] [Q&A]
Hi. First of all, thank you very much for the hard work!
I'm trying to use protomaps' basemap in MapLibre GL JS and using my own styles. I have specific use cases where for example show all roads in zoom levels like 2 or all "rail"s in zoom level 3. I found out by default such things come up in high zoom levels and therefore, it's not possible to set a minzoom and expect features to be shown.
Can you guys recommend a solution?

**Comment by nvkelso** [comment]
Thanks for the kind words :)

In cases like this we recommend creating a custom "thematic" tileset that includes your target low zoom features and load it as a 2nd tile source in your MapLibre style, and create custom style layers to draw these features "above" the default "reference" Protomaps styled features from the 1st tile source from protomaps.

Showing "all" roads or "all" rails at low-zooms can create extreme file size and visual hierarchy problems for a general purpose basemap, so they are excluded from low-zooms by default in Protomaps.
